src/components/ScrobbleItem.tsx

```diff
@@ -34,6 +34,7 @@ import { useSettings } from 'hooks/useSettings';
 
 interface ScrobbleItemProps {
   scrobble: Scrobble;
+  scrobbleRemovalPattern?: string;
```

Comment:
nit: could we rename this `cleanupPattern`?

Reply:
yes, nice suggestion